### ISSUE TITLE
Train script & dataloader updates for MoGe-2

### ISSUE THREAD

**Issue by karimknaebel** [microsoft/MoGe#69]
Hi, congrats on the release; yet another impressive model! Do you plan to update the training pipeline for MoGe-2? Currently there is no config (except for the one included in the checkpoint) and no normals dataloader and loss.

**Comment by zhoujiawei3** [comment]
@EasternJournalist Could you please let us know when the training code will be open‑sourced? I’m looking forward to trying it out soon.
